child-loader.mjs

```diff
@@ -1,8 +1,25 @@
-import { fileURLToPath } from 'url';
 import { createRequire } from 'module';
+import { fileURLToPath } from 'url';
+import { versionGteLt } from './dist/util.js';
+
 const require = createRequire(fileURLToPath(import.meta.url));
 
 // TODO why use require() here?  I think we can just `import`
 /** @type {import('./dist/child-loader')} */
 const childLoader = require('./dist/child/child-loader');
 export const { resolve, load, getFormat, transformSource } = childLoader;
+
+// On node v20, we cannot lateBind the hooks from outside the loader thread
+// so it has to be done here.
```

Comment:
Yeah, that's effectively what's happening here. It's calling bootstrap to late-bind the hooks with the state passed in on the loader URL.

I suppose it could get that information via the message port, but if we have the config we need on the URL, I'm not sure what that gets us?